docs/Localiser/index.md

```diff
@@ -0,0 +1,47 @@
+# Localiser
+A localising system for networked messages. 
```

Comment:
Better:
> A utility to construct multi-part localised strings which can also be sent across the network

Because the primary function of this is not the networking ... they are 90% as useful in local use only. Sending them via RPC is just a neat thing you can also do. We discussed this via DM, but I think the network bit could/should be its own mini util built on top of `Localiser`. Right now people are supposed to send their own raw ScriptRPC according to the docs, which feels meh.

Reply:
> they are 90% as useful in local use only

Judging from Expansions existing usecase for `StringLocaliser`, 70% in Server to Client RPC use. 30% local use.

> We discussed this via DM, but I think the network bit could/should be its own mini util built on top of Localiser. Right now people are supposed to send their own raw ScriptRPC according to the docs, which feels meh.

We can circle back to this after the new RPC is implemented.